lerna.json

```diff
@@ -4,5 +4,5 @@
     "packages/*"
   ],
   "useWorkspaces": true,
-  "version": "0.9.23"
+  "version": "0.9.24-alpha.0"
```

Comment:
?

Reply:
Had to be done to build the server to use in devnet.